ArticleTemplates/assets/js/bootstraps/football.js

```diff
@@ -69,7 +69,7 @@ function footballChart(homeTeam, awayTeam) {
         .attr('transform', d => {
             d.innerRadius = 0;
             d.outerRadius = radius;
-            let textpoint = arc.centroid(d);
+            let textpoint = arc().centroid(d);
```

Comment:
Should this be `arcShape`?

Reply:
Good spot!!! That fixed the visual bug ha! 🙈

Reply:
<img src="https://user-images.githubusercontent.com/19835654/94943917-5fafd680-04d0-11eb-86c1-8b5c021b1bfe.png" width="300px" />